assets/src/design-system/components/contextMenu/menu-item.js

```diff
@@ -0,0 +1,153 @@
+/*
+ * Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+/**
+ * External dependencies
+ */
+import PropTypes from 'prop-types';
+import { useMemo } from 'react';
+import styled, { css } from 'styled-components';
+
+/**
+ * Internal dependencies
+ */
+import { Button } from '../button';
+import { Link } from '../typography/link';
+import { Text } from '../typography/text';
+
+const reverseCss = ({ isLTR }) => css`
```

Comment:
Are these not aren't already handled by the `stylisRTLPlugin` ?

If you want to, you can just wrap your storybook instance in it and use the toggle to dictate it's application. will do the same thing it does in the editor:
```
import { StyleSheetManager, ThemeProvider } from 'styled-components';
import stylisRTLPlugin from 'stylis-plugin-rtl';
...
<StyleSheetManager stylisPlugins={isRTL ? [stylisRTLPlugin] : []}>
  // storybook component
</StyleSheetManager>
```

Reply:
Removed my extra styling, it's not needed!